dotnet/Dockerfile.tpl

```diff
@@ -0,0 +1,27 @@
+# AUTOGENERATED FILE
+FROM #{FROM}
+
+RUN [ "cross-build-start" ]
+
+RUN apt-get update && \
+    apt-get install -y curl libunwind8 gettext icu-devtools
+
+RUN rm -rf /tmp/* /var/tmp/* && \
+    rm -rf /var/lib/apt/lists/* && \
+    rm -f /var/cache/apt/archives/*.deb /var/cache/apt/archives/partial/*.deb /var/cache/apt/*.bin
+
+WORKDIR /tmp
+
+RUN curl -sSL -o dotnet.tar.gz #{DOTNET_BUNDLE}
+
+RUN mkdir -p /opt/dotnet && \
+    tar zxf dotnet.tar.gz -C /opt/dotnet && \
+    ln -s /opt/dotnet/dotnet /usr/local/bin
+
+RUN rm -rf /tmp/*
```

Comment:
Can we merge this RUN instruction with 2 previous ones so they just one layer is created instead of three?

Reply:
Sure